app/views/admin/adjustments/index.html.erb

```diff
@@ -42,3 +41,19 @@
     </table>
   </div>
 </div>
+
+<script>
+  $(document).ready(function () {
+      // will call refreshPartial every 15 seconds
+      setInterval(refreshPartial, 15000)
+
+  });
+
+  // calls action refreshing the partial
+  function refreshPartial() {
+    $.ajax({
+      url: '/admin/voteringar/refresh',
+      method: 'POST'
+   })
+  }
+</script>
```

Comment:
Now it is loaded as a resource and it checks for a div with `#vote-status`.

The above solution did not work, and therefore it took 2 hours until I realized how it could be fixed with a  single freakin `if-statement`...